inst/components/sidebar.scss

```diff
@@ -60,12 +60,11 @@ $bslib-sidebar-border: var(--bs-card-border-width, #{$card-border-width}) solid
   }
 
   > .collapse-toggle {
-    grid-row: 1 / 2;
     grid-column: 1 / 2;
```

Comment:
Is this change necessary?

Reply:
Done in b9ecbe649ac966716e530cd7cb67147a360bf3d7